nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/pom.xml

```diff
@@ -62,6 +62,12 @@
             <artifactId>caffeine</artifactId>
             <version>2.8.1</version>
         </dependency>
+        <dependency>
+            <groupId>antlr</groupId>
+            <artifactId>antlr</artifactId>
+            <version>2.7.7</version>
```

Comment:
This version of ANTLR is dated from 2007, is it possible to use a newer version?

Reply:
On further review, I noted that that jasn1-compiler uses this same version, so it makes sense to keep this version.